template/styles/sass/partials/_layout.scss

```diff
@@ -6,14 +6,13 @@
 }
 
 .body-content {
-    display: flex;
```

Comment:
Is there a reason these styles were moved?

Reply:
The flex layout of the body content was broken in IE 11 and it has no impact on the view of the page in the other browsers

Reply:
Why is it moved for a particular @media though?
https://github.com/IgniteUI/igniteui-docfx-template/pull/178/files#diff-95a4f2c06ae85035b926fc86d96ddc10R840

Seems to be applied to:
@media only screen and (max-width: 991px)

When this takes affect wont't it cause the layout be broken in IE if it cannot handle the flex styles?

Reply:
I removed it in this media query because, the layout on max screen widths below 991px should look like this:
![image](https://user-images.githubusercontent.com/25771040/79227355-6e45fb00-7e68-11ea-9f78-79d8ade33a3e.png)
It triggers the mobile view, so all the elements should be in a column layout.